hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/format/cow/CopyOnWriteInputFormat.java

```diff
@@ -130,7 +133,7 @@ public void open(FileInputSplit fileSplit) throws IOException {
     this.itr = RecordIterators.getParquetRecordIterator(
         internalSchemaManager,
         utcTimestamp,
-        true,
+        caseSensetive,
```

Comment:
Can you examplify an use case that `true` would got issues here?

Reply:
Here set true would not cause any problem，but when user table schema like （day，Day） and user did not want case-insensetive would not fit the requirement. This logic had handled in ParquetColumnarRowSplitReader##clipParquetSchema both for true or false，but not all called method set true fix，so I thought it should be configurable better. @danny0405

Reply:
> but when user table schema like （day，Day） and user did not want case-insensetive would not fit the requirement

Are you saying user needs the field names to be case sensitive?

Reply:
Actually some special case need to be true as for the schema check guard. @danny0405

Reply:
As mentioned above, please give an illustrative example here.

Reply:
OK，eg: 
1）When parquet real schema is (Day，day，user)， if caseSensitive is true，selected schema is (Day，day，user) would be push down to read parquet success  When build ParquetFileReader，this sence is OK；
2) When parquet real schema is (day，user)，but user selected schema is （dAy，UseR)，if caseSensitive is true it would not be push down to read parquet success；this sence should change caseSensitive to false to fit it. @danny0405

Reply:
For example，this could be used in unify sql engine open source or develop self by some business，could improve user sql such as unify casesensitivie or standard sql，this is a potential requirement. @danny0405

Reply:
> this is not user behavior.This is a potential requirement with change

But it seems the DDL and DML handle the case-sensitivity in different ways, which is not plausible here.

Reply:
Emm，but don't know the reason for handle false logic in ParquetFileReader.